web/src/common/api.service.js

```diff
@@ -69,6 +69,14 @@ export const CSVService = {
     return ApiService.download('csv', slug);
   },
 
+  downloadUrl(slug) {
```

Comment:
This function is never called.

Reply:
`download(slug) {` neither but it still there

Reply:
Then can we remove the dead code?